includes/amp-helper-functions.php

```diff
@@ -54,10 +54,23 @@ function amp_deactivate( $network_wide = false ) {
  * @since 1.5
  */
 function amp_bootstrap_plugin() {
+	/**
+	 * Filters whether AMP is enabled on the current site.
+	 *
+	 * Useful if the plugin is network activated and you want to turn it off on select sites.
+	 *
+	 * @since 0.2
+	 * @since 2.0 Postponed to run at plugins_loaded:8 instead of right after file load.
```

Comment:
The since tag needs to be adapted with the new timing:
```suggestion
	 * @since 2.0 Postponed to run at plugins_loaded:PHP_INT_MIN instead of right after file load.
```
Now wondering whether this is even comprehensible...

Reply:
Actually, “postponed” is the wrong word even. It should be “preponed” (which sadly is not a word).

```suggestion
	 * @since 2.0 Filter now runs at plugins_loaded (with earliest priority) rather than at the after_setup_theme action.
```

Reply:
https://www.dnaindia.com/lifestyle/report-now-entire-world-can-prepone-like-indians-only-1353503

Reply:
May it be adopted!